provisioning_client/tests/common_prov_e2e/prov_hsm/x509_msr.c

```diff
@@ -354,12 +354,12 @@ static int process_riot_key_info(X509_CERT_INFO* x509_info)
                 LogError("Failure: producing alias cert.");
                 result = MU_FAILURE;
             }
-            else if (generate_root_ca_info(x509_info, &tbs_sig) != 0)
+            else if (generate_root_ca_info(x509_info) != 0)
```

Comment:
This would suggest that in the previous comment, we will have a root CA key avail on the subsequent produce_device_cert call.

Reply:
Yep, fixed previous